modules/age.nix

```diff
@@ -261,44 +266,66 @@ in {
         }
       ];
     }
-
     (optionalAttrs (!isDarwin) {
+      # When using sysusers we no longer be started as an activation script
+      # because those are started in initrd while sysusers is started later.
+      systemd.services.agenix-install-secrets = mkIf sysusersEnabled {
+        wantedBy = ["sysinit.target"];
+        after = ["systemd-sysusers.service"];
+        unitConfig.DefaultDependencies = "no";
+
+        serviceConfig = {
+          Type = "oneshot";
+          ExecStart = pkgs.writeShellScript "agenix-install" (
```

Comment:
There seems to be an error because `mount` is not in `$PATH`.
`/nix/store/bsl27gh4hjybx6csgvhw7q99iqzlx0vw-agenix-install: line 8: mount: command not found`

Reply:
> `/nix/store/bsl27gh4hjybx6csgvhw7q99iqzlx0vw-agenix-install: line 8: mount: command not found`

Confirmed this issue exists. I didn't notice before since it doesn't causes unit fail but the `agenix.d` does not mount into ramfs.

Reply:
This should have been fixed now. Could you test it?

Reply:
Works fine, no error this time.